big_tests/dynamic_domains.config

```
@@ -131,6 +131,12 @@
         {host, <<"localhost">>},
         {password, <<"distributionftw">>},
         {port, 5232}]},
+    {clusterbuddy, [
+        {username, <<"clusterbuddy">>},
+        {server, <<"localhost">>},
```

Please use the dynamic domain - otherwise this user just tests the static one.
```suggestion
        {server, <<"domain.example.com">>},
```